azurerm/resource_arm_virtual_machine_unmanaged_disks_test.go

```diff
@@ -30,6 +30,196 @@ func TestAccAzureRMVirtualMachine_basicLinuxMachine(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMVirtualMachine_basicLinuxMachine_storageBlob_attach(t *testing.T) {
+	var vm compute.VirtualMachine
+	ri := acctest.RandInt()
+	preConfig := testAccAzureRMVirtualMachine_basicLinuxMachine(ri, testLocation())
+	prepConfig := fmt.Sprintf(testAccAzureRMVirtualMachine_basicLinuxMachine_destroyVM, ri, ri, ri, ri, ri)
+	config := fmt.Sprintf(testAccAzureRMVirtualMachine_basicLinuxMachine_storageBlob_attach, ri, ri, ri, ri, ri, ri, ri)
```

Comment:
would it be possible to update these to use the newer-style formatting methods? in particular we now pass in the test location, allowing these tests to be run in different regions. [Here's an example of what I'm referring too](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_virtual_machine_scale_set_test.go#L80) and [the associated formatting method](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_virtual_machine_scale_set_test.go#L1048)

Reply:
Sure! I'll update the PR shortly.